docs/fundamentals/toc.yml

```diff
@@ -2470,36 +2470,54 @@ items:
   items:
   - name: Overview
     href: ../core/porting/index.md
-  - name: .NET Upgrade Assistant tool
+  - name: General Information
     items:
-    - name: Overview
-      href: ../core/porting/upgrade-assistant-overview.md
-    - name: Telemetry
-      href: ../core/porting/upgrade-assistant-telemetry.md
-    - name: ASP.NET Core
-      href: ../core/porting/upgrade-assistant-aspnetmvc.md
-    - name: Windows Presentation Foundation
-      href: ../core/porting/upgrade-assistant-wpf-framework.md
-    - name: Windows Forms
-      href: ../core/porting/upgrade-assistant-winforms-framework.md
-  - name: Breaking changes
-    displayName: app compatibility
-    href: ../core/compatibility/
-  - name: Analyze third-party dependencies
-    href: ../core/porting/third-party-deps.md
-  - name: Port libraries
-    href: ../core/porting/libraries.md
-  - name: Organize projects for .NET
-    href: ../core/porting/project-structure.md
-  - name: Unavailable technologies
-    href: ../core/porting/net-framework-tech-unavailable.md?toc=/dotnet/fundamentals/toc.json&bc=/dotnet/breadcrumb/toc.json
-  - name: Use the Windows Compatibility Pack
-    href: ../core/porting/windows-compat-pack.md
-  - name: Port Windows Forms projects
-    href: /dotnet/desktop/winforms/migration/?view=netdesktop-5.0
-  - name: Port WPF projects
-    href: /dotnet/desktop/wpf/migration/convert-project-from-net-framework?view=netdesktop-5.0
-  - name: Port C++/CLI projects
-    href: ../core/porting/cpp-cli.md
-  - name: Choose between .NET 5 and .NET Framework for server apps
-    href: ../standard/choosing-core-framework-server.md
+    - name: About .NET
+      items:
+      - name: Versioning info for .NET SDK, MSBuild, and Visual Studio
+        href: ../core/porting/versioning-sdk-msbuild-vs.md
+      - name: Choose between .NET 5 and .NET Framework for server apps
+        href: ../standard/choosing-core-framework-server.md
+    - name: .NET Upgrade Assistant tool
+      items:
+      - name: Overview
+        href: ../core/porting/upgrade-assistant-overview.md
+      - name: Windows Presentation Foundation
+        href: ../core/porting/upgrade-assistant-wpf-framework.md
+      - name: Windows Forms
+        href: ../core/porting/upgrade-assistant-winforms-framework.md
+      - name: ASP.NET Core
+        href: ../core/porting/upgrade-assistant-aspnetmvc.md
+    - name: Breaking changes
+      displayName: app compatibility
+      href: ../core/porting/breaking-changes.md
```

Comment:
@adegeo @StephenBonikowsky What was the inspiration behind replacing the link to the /core/compatibility TOC with a link to a new /core/porting/breaking-changes.md article? This effectively orphans the /core/compatibility [TOC](https://learn.microsoft.com/en-us/dotnet/core/compatibility/).

Reply:
By TOC do you mean this "Types of compatibility" page? (https://learn.microsoft.com/en-us/dotnet/core/compatibility/categories)

If so, we still link to it from https://learn.microsoft.com/en-us/dotnet/core/porting/breaking-changes#types-of-compatibility

A lot changed so I can't give you reasoning for that specific change. There is a lot more content desired for the whole 'Migration guide' section but priorities and resourcing hasn't permitted getting back to it.

Reply:
Sorry, I meant "table of contents". The "compatibility" table of contents is separate to the main .NET table of contents, and so it needs to be linked from the main table of contents. But by understanding the thinking behind your change of linking to a new article, I can perhaps find a different place to link to it.

Reply:
@gewarren If I remember correctly, it was to too much open information, versus the guided philosophy of the new migration guide. Instead, a new "breaking changes" page for the migration guide was created to give context FIRST, and then if they desired, they can check out the breaking changes TOC. This way, they're not overwhelmed with "here are the 1000 types of breaking things you'll have to watch out for" which may sway them away from migrating off of .NET Framework.